<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolStatistic</class>
 <widget class="QWidget" name="ToolStatistic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>673</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBox_ValidValue">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>48</y>
     <width>103</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>有效值范围</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>224</width>
     <height>171</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>229</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>上限</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_UpLimit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>40</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_DownLimit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>120</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>下限</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>662</width>
     <height>654</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_Task">
        <property name="text">
         <string>选择统计任务</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_Task">
        <property name="minimumSize">
         <size>
          <width>211</width>
          <height>621</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>211</width>
          <height>621</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QListWidget" name="listWidget_Task">
           <property name="minimumSize">
            <size>
             <width>191</width>
             <height>601</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>191</width>
             <height>601</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_Result">
          <property name="text">
           <string>选择统计结果</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_Result">
          <property name="minimumSize">
           <size>
            <width>211</width>
            <height>621</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>211</width>
            <height>621</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QListWidget" name="listWidget_Result">
             <property name="minimumSize">
              <size>
               <width>191</width>
               <height>601</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>191</width>
               <height>601</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_Data">
          <property name="text">
           <string>选择统计数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_Data">
          <property name="minimumSize">
           <size>
            <width>211</width>
            <height>621</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>211</width>
            <height>621</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QListWidget" name="listWidget_Data">
             <property name="minimumSize">
              <size>
               <width>191</width>
               <height>601</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>191</width>
               <height>601</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
